### PR TITLE
Nicolehaug devpy

### PR DESCRIPTION
**Pull request by nicolehaug**
Made change to dev.py and eb.py to allow hostname http://ga-dev.fs2c.usda.gov/

### CODE REVIEW

**Comment by nicolehaug** [comment]
ps @beechnut I may not have access to merge, so if approved, go ahead and merge if you would. Thanks!

**Review by beechnut — APPROVED** [review]
@nicolehaug Just a note that I squashed these commits before merging, which is why there's only one now. Thanks for this, approving and merging!

**Pull request merged**
